backend/serverless.yml

```diff
@@ -51,10 +77,10 @@ functions:
       - websocket:
           route: $connect
           # Only if we want authenticated connections
-          authorizer:
-            name: auth
-            identitySource:
-              - 'route.request.header.Auth'
+          # authorizer:
+          #   name: auth
+          #   identitySource:
+          #     - 'route.request.header.Auth'
```

Comment:
problem with this ?

Reply:
It's not useful and will cost lambda invocation quota

Reply:
then delete it ?